templates/JSON/treat_ad.json

```diff
@@ -0,0 +1,280 @@
+{
+    "name": "TREAD_AD_YAML",
+    "type": "Project",
+    "children": [
```

Comment:
I guess I didn't approve this right away because of the conversation that was happening on [Slack](https://sagebionetworks.slack.com/archives/G01KHN0D1V2/p1618362260063300). 

IHMO, the expanded JSON should allow for multiple resources (_e.g._ projects, users/teams eventually) in the root. This can be achieved using either a mapping or an array. I'm partial to mappings because they look nicer in YAML and the keys can be used down the road for cross-references. 

I can be convinced otherwise, but I haven't seen a compelling reason against this chance.

Reply:
Users/Teams == "Project Sharing Settings" in the UI?

Reply:
By users/teams, I'm referring to the creation of teams for a project. Admittedly, we probably wouldn't create users because that needs to be done by a human (including the certification process). I realize this is a feature tabled for later. I think it would be wise to design the expanded JSON template to accommodate multiple top-level resources. 

Setting users/teams aside, I think a template should be able to express a multi-project setup. This might not be part of our MVP, but again, I think we would save ourselves the headache of having to update the code by making this simple tweak upfront. 

Another way of putting it, why not nest top-level resources under a JSON mapping or array?

Reply:
I thought teams existed independent of projects.  Talk me through why you think they belong in the same template, please?  I think that's the disconnect I'm having with your idea - they're OOS for this regardless, but I'd like to understand you.  I thought you were talking about ACLs here, which should be next in our list.

I have a note from one of our calls that we wouldn't attempt to address the use-case of creating multiple projects in the same template - and we couldn't figure out what extracting the structure of anything above a project could look like.  Other than that, I agree that the template needs to support multiple resources.  It will be required for ACLs anyway.

Reply:
From what I recall, we decided to start with single-project templates for simplicity and to avoid cross-references for now. I wasn't thinking of restricting the template format to one root resource per template. You can ask what the others remember from that conversation on Slack in case I'm mistaken. 

I also recall us discussing a two-pass approach, _i.e._ creating all teams first and then projects/folders with their ACLs. This would be implemented after the MVP, but I don't see why our template format couldn't accommodate projects/folders and teams in the same file. For example, if my stack only had one project and one team, I wouldn't want two super short files; I would prefer everything being in one file (especially when we'll support cross-references). 

To clarify, I'm just trying to advocate for small tweaks to our template format that will make it more future-proof, especially if the tweaks have no downsides. The actual implementation can wait based on what we've planned.

Reply:
Agreed with Bruno.

Reply:
Bruno, I'm not sure it's clear that I'm agreeing with you, and in no way disputing that the template should support different resources.

I provided a template for a project and its folders (working on a story called "Projects and Folders"); I agree that indexing the resources in an array is useful for representing multiple resources, and that's what I changed my template to reflect (still pending review).

PS: we have a confluence page for the project where we can document meetings, design decisions, brainstorms, etc.

Reply:
Thanks @mattfazza , can there be a PR to change the yaml template as well? this LGTM

Reply:
I was prompted by this question of yours:
> Talk me through why you think they belong in the same template, please?

I did notice the change on the template but forgot to approve the PR.